client/src/components/ratings/Ratings.jsx

```diff
@@ -1,13 +1,37 @@
 import React, { useState, useEffect } from 'react';
 import fetcher from '../../fetchers';
+import ReviewTile from './sub-components/ReviewTile.jsx';
+import ReviewDashboard from './sub-components/ReviewDashboard.jsx';
+import axios from 'axios';
 
 export default function Ratings({
   feature
 }) {
 
+  const [reviews, setReviews] = useState([]);
+
+  useEffect(() => {
+    fetcher.ratings.getReviews(40348)
```

Comment:
should get reviews for featured product via `feature.id`

Reply:
Got it - used this as a placeholder until we get id from top level of app